src/uct/ib/base/ib_device.h

```diff
@@ -271,6 +272,14 @@ ucs_status_t uct_ib_device_select_gid(uct_ib_device_t *dev,
 const char *uct_ib_device_name(uct_ib_device_t *dev);
 
 
+/**
+ * For the given IB device find the associated bus information
+ *
+ * @param [in]  dev             IB device.
+ * @param [out] bus_id          Bus information
```

Comment:
minor: missing `.` in the end (or can remove it in the end of the previous line)